launchcontainers/createsymlinks.py

```diff
@@ -123,17 +123,30 @@ def check_tractparam(lc_config, sub, ses, tractparam_df):
     container = lc_config["general"]["container"]
     precontainer_anat = lc_config["container_specific"][container]["precontainer_anat"]
     anat_analysis_num = lc_config["container_specific"][container]["anat_analysis_num"]
-    
-    fs_zip = os.path.join(
-        basedir,
-        "BIDS",
-        "derivatives",
-        precontainer_anat,
-        "analysis-" + anat_analysis_num,
-        "sub-" + sub,
-        "ses-" + ses,
-        "output", "fs.zip"
-    )
+    if run_lc:
+        rtp_analysis_num = lc_config["general"]["analysis_name"]
+        fs_zip = os.path.join(
+            basedir,
+            "BIDS",
+            "derivatives",
+            container,
```

Comment:
this is not in line with the design of anatrois. if you want to read the preanalysisfs you need to change in the session lc_config['container_specific']['anatrois'] and you define the occasion of fs_zip.  run_lc shouldn't control the behaviour of a container specific feature

Reply:
I would say this change is too specific... it seems only works for your situation at that time, and it will always work if you input the info in the `lc_config['container_specific']['anatrois'] preanalysisfs option`